index.js

```diff
@@ -14,7 +15,10 @@ function startServer() {
     if (!module.parent) {
     // listen on port config.port
         app.listen(config.port, () => {
-            debug(`server started on port ${config.port} (${config.env})`);
+            logger.info({
+                service: 'messaging-service',
+                message: `server started on port ${config.port} (${config.env})`,
```

Comment:
**Non-blocking**
`config.port` and `config.env` are perfect candidates for log keys. Agnostic about punting to https://jira.amida.com/browse/ORANGE-795 when the log format will be updated anyways.